portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,3 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
```

Comment:
We might want to keep this around because I believe we might be doing some javascript next week.